src/content/en/fundamentals/web-app-manifest/index.md

```diff
@@ -245,6 +250,11 @@ but you can provide additional icons as necessary.
 
 <div class="clearfix"></div>
 
+## Generating manifests (and icons) automatically
```

Comment:
Please remove this section. PWABuilder still has a bunch of issues that I'd rather not recommend people use it.